tests/ADService.test.js

```diff
@@ -235,6 +290,7 @@ describe.only("ADService", () => {
     });
 
     test("calls secureStore.setItemAsync with correct params", async () => {
+      expect(props.setTokenState).toHaveBeenCalledTimes(1);
```

Comment:
Don't really understand why this isn't reset before the test? Also tried reinitating the adService with no luck.

If the test is run seperately it succeed in the old version. but for some reason the new test introduced inteferes with this one. Maybe you have an idea of what's going on?